crates/circuits/sha256-air/src/trace.rs

```diff
@@ -50,7 +50,7 @@ impl Sha256Air {
         trace_width: usize,
         trace_start_col: usize,
         input: &[u32; SHA256_BLOCK_WORDS],
-        bitwise_lookup_chip: &BitwiseOperationLookupChip<8>,
+        bitwise_lookup_chip: SharedBitwiseOperationLookupChip<8>,
```

Comment:
why are we doing this...? it doesn't really matter, but cloning `Arc` is worse than `&`. 
You could also make `SharedBitwiseOperationLookupChip` implement `Borrow` I think?